### ISSUE TITLE
Register the affined image to MNI 1mm template, shapes mismatch

### ISSUE THREAD

**Issue by JinFeng-Wu** [voxelmorph/voxelmorph#487]
I'm using this:
scripts/tf/register.py --moving t1_mni.nii.gz --fixed MNI152_T1_1mm_brain.nii.gz --moved t1_mni_voxelmorph.nii.gz --model vxm_dense_brain_T1_3D_mse.h5 

t1_mni.nii.gz is a affined image to MNI template. The image size is 182 X 218 X 182.

Error produced:
ValueError: A `Concatenate` layer requires inputs with matching shapes except for the concatenation axis. Received: input_shape=[(None, 22, 26, 22, 32), (None, 22, 27, 22, 32)]

Do you have any idea why this happened? Thank you.

**Comment by adalca** [comment]
Yup this sort of an image happens when the data you are trying to register is not a multiple of 32. Can you pad them to a multiple of 32 and see if this solves it?

**Comment by JinFeng-Wu** [comment]
Thank you, it worked by padding the image. But the output warped image was a empty image, no matter what kind of pretrained models I specified, I don't know why.

Command & log:
/Users/michaelwu/Downloads/voxelmorph-dev/scripts/tf/register.py --moving t1_mni_norm.nii.gz --fixed mni_1mm_brain_resam.nii.gz --moved t1_mni_morph.nii.gz --model vxm_dense_brain_T1_3D_mse.h5 --warp t1_mni_warp.nii.gz
2022-11-11 10:05:44.291040: I tensorflow/core/platform/cpu_feature_guard.cc:193] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
2022-11-11 10:05:48.784009: I tensorflow/core/platform/cpu_feature_guard.cc:193] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 FMA
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
/Users/michaelwu/opt/anaconda3/lib/python3.9/site-packages/voxelmorph-0.2-py3.9.egg/voxelmorph/tf/networks.py:125: UserWarning: int_downsize is deprecated, use the int_resolution parameter.
/Users/michaelwu/opt/anaconda3/lib/python3.9/site-packages/voxelmorph-0.2-py3.9.egg/voxelmorph/tf/networks.py:130: UserWarning: unet_half_res is deprecated, use the svf_resolution parameter.
1/1 [==============================] - 7s 7s/step
1/1 [==============================] - 1s 1s/step
